views/items/index.pug

```diff
@@ -15,6 +15,11 @@ mixin display( nodes, depth )
         +display( entry.children, depth + 1 )
 
 block content
+
+  #myModal.modal.fade(tabindex='-1', role='dialog', aria-labelledby='myModalLabel', aria-hidden='true')
```

Comment:
Please provide a more meaningful ID than `myModal` - maybe `helpModal`?  The classes contained in this block should also be named more meaningfully.